frontend/src/store/slices/auth/auth.slice.ts

```diff
@@ -0,0 +1,35 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { ReducerName } from 'common/enums';
+import { AppThunk } from 'common/types';
+import { AuthState } from 'common/types/app';
+import { IUserLoginPayload } from 'healthcare-shared/common/interfaces';
+import { authApi } from 'services';
+
+const initialState: AuthState = {
+  user: {},
+  isAuthorized: false,
+  isLoading: true,
+};
+
+const { reducer, actions } = createSlice({
+  name: ReducerName.AUTH,
+  initialState,
+  reducers: {
+    //TODO: create method that will dispatch in login
+  },
+});
+
+const setToken = (token: string) => localStorage.setItem('token', token);
```

Comment:
It would be better to have a service for storage. Something like this: 
```
// frontend/services/services.ts
import { Storage } from './storage/storage.service.ts'

const storage = new Storage({
  storage: localStorage
});

export { storage }

// frontend/services/storage/storage.service.ts
class Storage {
  constructor({ storage }) {
    this._storage = storage
  }

  getItem(key) {
    return this._storage.getItem(key)
  }

  setItem(key, value) {
    return this._storage.setItem(key, value)
  }

  removeItem(key) {
    return this._storage.removeItem(key)
  }

  clear() {
    return this._storage.clear()
  }
}

export { Storage }

```